test/WebJobs.Extensions.DurableTask.Analyzers.Test/Orchestrator/TimerAnalyzerTests.cs

```diff
@@ -269,6 +301,390 @@ public static string DirectCall()
             VerifyCSharpDiagnostic(test, expectedResults);
         }
 
+        [TestMethod]
+        public void CodeFixProvider_Await_New()
+        {
+            var test = @"
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.WebJobs;
+
+    namespace VSSample
+    {
+        public static class HelloSequence
+        {
+            [FunctionName('E1_HelloSequence')]
+            public static async Task<List<string>> Run(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+            {
+                await Task.Delay(new TimeSpan(100), new CancellationToken());
+            }
+        }
+    }";
+
+            var fix = @"
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.WebJobs;
+
+    namespace VSSample
+    {
+        public static class HelloSequence
+        {
+            [FunctionName('E1_HelloSequence')]
+            public static async Task<List<string>> Run(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+            {
+            await context.CreateTimer(context.CurrentUtcDateTime.Add(new TimeSpan(100)), new CancellationToken());
```

Comment:
Shouldn't there be 4 more spaces before the `await` symbol to match the original source code?